Signal/src/call/PeerConnectionClient.swift

```diff
@@ -1092,6 +1110,90 @@ class HardenedRTCSessionDescription {
     }
 }
 
+protocol VideoCaptureSettingsDelegate: class {
+    var videoWidth: Int32 { get }
+    var videoHeight: Int32 { get }
+}
+
+class VideoCaptureController {
```

Comment:
This is based on the example class from upstream: https://webrtc.googlesource.com/src/+/c522e758f6a364c23e7232d4f402713601f0b35e%5E%21/